hubblestack/daemon.py

```diff
@@ -604,10 +604,6 @@ def __init__(self, message, levelname, asctime, name):
 # pick it up again.  If the hang is transient, the grain will populate
 # normally.
 #
-# repeats=True meaning: restart the signal itimer after firing the timeout
-# exception, which salt catches. In this way, we can catch multiple hangs with
-# a single timer. Each timer restart is a new 600s timeout.
-#
 # tag='hubble:rg' will appear in the logs to differentiate this from other
```

Comment:
This is partially a lie. Normally the timeout exceptions will be caught by salt's try:except blocks internal to (eg) grains.core. But you still get the stopit log.warning() if nothing else.